django_stomp/services/consumer.py

```diff
@@ -3,18 +3,23 @@
 import ssl
 import time
 import uuid
+from concurrent.futures import ThreadPoolExecutor
 from dataclasses import dataclass
 from enum import Enum
 from typing import Callable
 from typing import Dict
+from typing import Optional
 
 import stomp
+from django.conf import settings
 from django_stomp.helpers import create_dlq_destination_from_another_destination
 from django_stomp.helpers import only_destination_name
 from stomp import connect
 
 logger = logging.getLogger("django_stomp")
 
+STOMP_PROCESS_MSG_WORKERS = getattr(settings, "STOMP_PROCESS_MSG_WORKERS", None)
```

Comment:
Maybe we can assert that the provided value is `int` and only raise an exception with a specific exception to inform the user otherwise.

Reply:
Yep! Clearly forgot to add some treatment here

Reply:
Added the treatment on [5fc1ea9](https://github.com/juntossomosmais/django-stomp/pull/21/commits/5fc1ea9a94c542713666ab09b0abcde7925f693d)